examples/python/images/image_reader_file.py

```diff
@@ -0,0 +1,55 @@
+###############################################################################
+# (c) 2005-2015 Copyright, Real-Time Innovations.  All rights reserved.       #
+# No duplications, whole or partial, manual or electronic, may be made        #
+# without express written permission.  Any such copies, or revisions thereof, #
+# must display this notice unaltered.                                         #
+# This code contains trade secrets of Real-Time Innovations, Inc.             #
+###############################################################################
+
+"""Reads images and saves them into a file"""
+
+from __future__ import print_function
+from sys import path as sys_path, stdout
+from os import path as os_path
+import matplotlib.pyplot as plot
```

Comment:
Works in python3, fails in python2:
(python2_env) sam@rti-10678:~/working/trees/connector/connector-py$ python examples/python/images/image_reader_file.py 
Traceback (most recent call last):
  File "examples/python/images/image_reader_file.py", line 14, in <module>
    import matplotlib.pyplot as plot
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/pyplot.py", line 115, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/backends/__init__.py", line 63, in pylab_setup
    [backend_name], 0)
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 4, in <module>
    from . import tkagg  # Paint image to Tk photo blitter extension.
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/backends/tkagg.py", line 5, in <module>
    from six.moves import tkinter as Tk
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/six.py", line 203, in load_module
    mod = mod._resolve()
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/six.py", line 115, in _resolve
    return _import_module(self.mod)
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/six.py", line 82, in _import_module
    __import__(name)
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 39, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ImportError: No module named _tkinter

Reply:
It worked for me, but I'll try again

Reply:
On linux with python 2.7 it doesn't work for me, but with a different error. On Mac it does work with both 3 and 2, but in 2 I had to explicitly add a line of code suggested by an error in the output.

I will just say that this example requires python 3.